lib/di/provider/dio_provider.dart

```diff
@@ -31,8 +33,8 @@ class DioProvider {
     }
 
     return dio
-      ..options.connectTimeout = 3000 as Duration?
-      ..options.receiveTimeout = 5000 as Duration?
+      ..options.connectTimeout = const Duration(milliseconds: 30000)
+      ..options.receiveTimeout = const Duration(milliseconds: 50000)
```

Comment:
Same here

Reply:
Resolved in https://github.com/nimblehq/ic-flutter-avishek/pull/28/commits/55417b71c60b16af19eb96ca7bb0016f46388dc0.